access/ms-teams/msapi/bot_framework_client.go

```diff
@@ -0,0 +1,135 @@
+package msapi
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gravitational/trace"
+)
+
+const (
+	botFrameworkAuthScope     = "https://api.botframework.com/.default"
+	botFrameworkBaseURL       = "https://smba.trafficmanager.net"
+	botFrameworkDefaultRegion = "uk"
```

Comment:
Set it to "us" by default?

Reply:
Unfortinately, there's no "us", and I have failed to find the region codes list. Moreover, according to my findings in the github issues, etc. bot framework is not working with all regions. So, I set this to `emea`, which seems to be the ms default region.